heart/test/ionicmodels/TestCodegenLong/TestCodegenLongBackwardEulerOpt.hpp

```diff
@@ -61,17 +61,19 @@ class TestCodegenLongBackwardEulerOpt : public CodegenLongHelperTestSuite
         args.push_back("--opt");
 
         // Models that need a smaller dt
-        std::vector<std::string> diff_models = spectail_streatment_models(models, {"iyer_model_2004",
+        std::vector<std::string> diff_models = special_treatment_models(models, {"iyer_model_2004",
                                                                                    "iyer_model_2007",
                                                                                    "jafri_rice_winslow_model_1998",
                                                                                    "pandit_model_2001_epi",
                                                                                    "priebe_beuckelmann_model_1998",
                                                                                    "viswanathan_model_1999_epi",
                                                                                    "winslow_model_1999"});
         // These have NaN in the jacobian due to massive exponentials
-        std::vector<std::string> bad_models = spectail_streatment_models(models, {"hund_rudy_2004_a"});
+        std::vector<std::string> bad_models = special_treatment_models(models, {"hund_rudy_2004_a"});
 
         HeartConfig::Instance()->SetOdePdeAndPrintingTimeSteps(0.01, 0.1, 1.0);
+        TS_ASSERT_THROWS_ANYTHING(RunTests(dirname, {"negative_concentration_paci_hyttinen_aaltosetala_severi_ventricularVersion"}, args));
```

Comment:
`TS_ASSERT_THROWS_CONTAINS` would be a bit nicer - checks that the exception is the one we are expecting